employees/models.py

```diff
@@ -0,0 +1,43 @@
+from django.db import models
+
+
+class BaseModel(models.Model):
```

Comment:
No need to declare separate model for this. we can use django provided model from django.extensions `TimeStampedModel`